### PR TITLE
Feature/rubocop

### PR DESCRIPTION
**Pull request by hez**
Rubocop 👮 👮‍♀ 

For discussion, what else do we want to enable? Change?

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+AllCops:
+  Exclude:
```
Comment (hez) [inline .rubocop.yml]: These files probably don't need linting.

**Comment by rfb** [comment]
@hez - I'm indifferent about what's enabled/disabled, just constancy!

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    - config/initializers/*
+    - config.ru
+    - db/default/*
+    - db/migrate/*
+    - db/schema.rb
+    - node_modules/**/*
+    - spec/rails_helper.rb
+    - spec/spec_helper.rb
+
+# We don't actually care about what string literals we use, both are good
```
Comment (hez) [inline .rubocop.yml]: As far as I know this just isn't a thing or worth worrying about. Thoughts?

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Style/StringLiterals:
+  Enabled: false
+
+Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
```
Comment (hez) [inline .rubocop.yml]: I figured we probably didn't need these on tests, don't see how it would get us much.

**Pull request merged**